py3status/modules/arch_updates.py

```diff
@@ -116,7 +120,7 @@ def _check_aur_updates(self):
 
         pending_updates = b""
         try:
-            pending_updates = str(subprocess.check_output(["cower", "-u"]))
+            pending_updates = str(subprocess.check_output(self.check_aur_cmd, shell=True))
```

Comment:
(Imo) This should be replaced with a py3 helper `self.py3.command_output()` first.